src/containers/Header/Search.jsx

```diff
@@ -73,15 +73,14 @@ const normalize = (id, type) => {
 class Search extends Component {
   constructor(props) {
     super(props);
-    this.state = { redirect: '', prevProps: this.props };
+    this.state = { redirect: '' };
     this.onKeyDown = this.onKeyDown.bind(this);
   }
 
-  static getDerivedStateFromProps(nextProps, prevState) {
-    if (nextProps.redirect !== prevState.prevProps.redirect) {
-      return { redirect: '', prevProps: nextProps };
-    }
-    return null;
+  /* eslint-disable */
+  UNSAFE_componentWillReceiveProps(nextProps) {
+    /* eslint-enable */
```

Comment:
There's an eslint command that will only disable for the next line, it's something like `disable-next-line`. It would be better to use that.

Reply:
The advantage of this is that it'll only disable the line for that specific rule, instead of for everything, which is what you're doing here